conda/core/link.py

```diff
@@ -1385,6 +1386,18 @@ def add_double(display_key, left_str, right_str):
                     left_str,
                     f"{right_str} {' '.join(link_prec.metadata)}",
                 )
+
+        if change_report.revised_precs:
+            builder.append("\nThe following packages will be REVISED:\n")
+            for namekey in sorted(change_report.revised_precs, key=convert_namekey):
+                unlink_prec, link_prec = change_report.revised_precs[namekey]
+                left_str, right_str = diff_strs(unlink_prec, link_prec)
+                add_double(
+                    strip_global(namekey),
+                    left_str,
+                    f"{right_str} {' '.join(link_prec.metadata)}",
+                )
```

Comment:
This block repeats across all sections so it's a very low hanging fruit to abstract this away to a helper function with the necessary parameters, but well, no need here.

Reply:
Don't tempt me with a good time! Pushed a small refactor.